app/controllers/api/projects.php

```diff
@@ -95,6 +95,7 @@
         $backups['database_db_fra1_03'] = ['from' => '10:30', 'to' => '11:15'];
         $backups['database_db_fra1_04'] = ['from' => '13:30', 'to' => '14:15'];
         $backups['database_db_fra1_05'] = ['from' => '4:30', 'to' => '5:15'];
+        $backups['database_db_fra1_06'] = ['from' => '16:30', 'to' => '17:15'];
```

Comment:
How is this related to usage PR? it seems like you got to branches mixed.

Reply:
It was intently.
I was comparing against usage from the cloud branch
and noticed that a db was added I thought it should be updated too.